### DISCUSSION TITLE
Shared my .git folder by mistake

### DISCUSSION THREAD

**Discussion by thenithinbalaji** [community/community#21175] [Other Feature Feedback, Questions, & Ideas] [answered]
I shared my .git folder by mistake. Will someone be able to push to my repo?

**Comment by Abhishek10351** [comment]
You can just gitignore it, it just contains the commit files and it is shared anyway.

**Comment by DenverCoder1** [comment, marked as answer]
Sharing the `.git` folder does not give push access to a repository.

They may be able to know the history of changes including loose changes that you changed your mind about.

There also may be some sensitive information in the `.git/config` such as usernames, emails, remote URLs. You can check the file to see all that was shared.
